pages/administratorCourseRequests/administratorCourseRequests.sql

```diff
@@ -23,8 +23,8 @@ WITH
         )
       ) AS jobs
     FROM
-      job_sequences AS js
-      LEFT JOIN course_requests AS cr ON cr.id = js.course_request_id
+      course_requests AS cr
+      JOIN job_sequences AS js ON cr.id = js.course_request_id
```

Comment:
While much better, this new query is still doing a seq scan on `job_sequences` because we don't have an index on `job_sequences.course_request_id`. As the size of the `job_sequences` table grows, this will eventually because a bad problem. Want to add an index now?

Reply:
Yeah, I think that's reasonable!

Tangential thought: not that it really matters, but in future situations like this, might it be better to have a table like `course_request_job_sequences` to join them instead of having `course_request_id` leaking into our generic job sequence table?

Reply:
Yep, for sure, that would be a better pattern.